tutordiscovery/plugin.py

```diff
@@ -115,3 +133,12 @@ def _mount_course_discovery(mounts, name):
 tutor_hooks.Filters.CONFIG_OVERRIDES.add_items(
     list(config.get("overrides", {}).items())
 )
+
+
+@tutor_hooks.Filters.APP_PUBLIC_HOSTS.add()
+def _print_discovery_public_hosts(hosts: list[str], context_name: t.Literal["local", "dev"]) -> list[str]:
+    if context_name == "dev":
+        hosts += ["discovery.{{ LMS_HOST }}:8381"]
```

Comment:
Replace "discovery.{{ LMS_HOST }}" by "{{ DISCOVERY_HOST }}" (on this line and the next one)